Main.java

```diff
@@ -3,12 +3,12 @@
 public class Main {
     public static void main(String[] args) {
         Scanner sc = new Scanner(System.in);
-        System.out.println("Name?");
+        System.out.println("What is the name of the course?");
         String name = sc.next();
-        System.out.println("Duration?");
+        System.out.println("How many hours?");
```

Comment:
hour is uncountablem, so it should be much

Reply:
UR stupid https://socratic.org/questions/is-hour-a-countable-noun-is-it-an-abstract-noun#:~:text=Explanation%3A,of%20a%20unit%20of%20time.

Reply:
Oh, my bad.